app/views/users/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Users</h1>
-<p>Here is a list of all users:</p>
\ No newline at end of file
+<div class="users">
+  <% @users.each do |user| %>
+    <%= link_to user do %>
+      <div class="user">
+        <div class="user-image">
+          <%= image_tag user.photo %>
+        </div>
+        <div class="user-info">
+          <h1><%= user.name %></h1>
+          <h3>Number of posts: <%= user.posts_counter %></h3>
+        </div>
```

Comment:
![image](https://github.com/joramot/BlogApp/assets/93039359/fa2365c0-37d9-4862-9ac3-0dba98ed8a57)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your posts counter work to make your app work as expected :+1: .